### ISSUE TITLE
shared dependencies

### ISSUE THREAD

**Issue by pcompassion** [sergei-maertens/django-systemjs#16]
I'd like to exclude jquery, underscore, bootstrap, etc from bundle.  

Those libraries could be bundled seperately as shared dependencies.  
And I believe the following is the jspm's answer to the problem.
(I don't get the syntax, hard to find doc explaining what it means excatly :( )

https://github.com/jspm/jspm-cli/issues/133 
https://github.com/jspm/jspm-cli/issues/872 mentions
`jspm bundle app/**/* - [app/**/*] dependency-bundle.js --inject` 

Would it be possible to create a shared dependency bundle with django-systemjs?  
(I'm not even sure the above method would work with django setup because django doesn't have a parent directory such as `app`)

**Comment by sergei-maertens** [comment]
Yes, this will be possible in future versions, but not currently. Ideally
all jspm options/cli args should/will be supported.

**Comment by pcompassion** [comment]
Would it be possible to manually integrate the above step with django-systemjs at the moment?

**Comment by sergei-maertens** [comment]
I don't think so, maybe there would be some workarounds by injecting bundle
config in the configfile, but I'm afraid there are a lot of edge cases to
take care of. I would recommend waiting until it's implemented.
